src/api/spec/support/shared_examples/features/beta/user_tab.rb

```diff
@@ -14,11 +14,30 @@ def toggle_checkbox(html_id)
     # Thus we have to remove bootstraps custom-control classes.
     page.execute_script("$('##{html_id}').removeClass('custom-control-input')")
     page.execute_script("$('label[for=#{html_id}]').removeClass('custom-control-label')")
+    scroll_page(html_id) if mobile?
     find_field(html_id).click
     # FIXME: Needed to wait for the Ajax call to perform
     sleep(1)
   end
 
+  def uncollapse_datatable_row(html_id, text)
```

Comment:
You invented `skip_on_mobile`. This is the perfect opportunity to make use of it :smile:

Reply:
As we were working in the tests, we skipped some tests only for mobile with this method.

Later on we were capable, with this workaround, to test for mobile, and not use the `skip_on_mobile` method at all. Right, we should not introduce a method that is not used.

But, do you prefer to get rid of this workaround, use the `skip_on_mobile`, and don't test this features for mobile?

Reply:
Don't test this on mobile, it's not worth it...

Reply:
Ok. Tests skipped on mobile.